flow-typed/box-ui-elements.js

```diff
@@ -618,6 +618,104 @@ type Tasks = {
     total_count: number,
 };
 
+/* New Task Types START */
+
+type ID = string;
+type ISODate = string;
+
+type UserMini = User;
+
+type FileMini = {
+    id: ID,
+    name: string,
+    type: 'file',
+};
+
+type FolderMini = {
+    id: ID,
+    name: string,
+    type: 'folder',
+};
+
+type TaskStatus = 'NOT_STARTED' | 'IN_PROGRESS' | 'COMPLETED' | 'APPROVED' | 'REJECTED';
```

Comment:
Normally we try to keep the actual re-usable constants in the constants.js file so that the strings are all in one place. And just use the imported constant here via typeof